src/client/components/lazy-chat/lazy-chat.vue

```diff
@@ -47,6 +47,16 @@ export default {
     onLoaded () {
       this.isLoaded = true
       this.provider.onLoaded()
+    },
+    handleClick () {
+      this.isRequested = true
+      this.track()
+    },
+    track () {
```

Comment:
To be consistent I'd set the `eventAction` to `click` and the `eventLabel` to `start`?
And actually, I'd change the signature of this method to `track(eventLabel)` like you do in `opt-in`. That way you can use it in `handleClick` and also `onOpened` and `onClosed` as suggested in my comment in `providers/crisp.js`